src/Export/ProductService.php

```diff
@@ -26,33 +28,45 @@ class ProductService
 {
     public const CONTAINER_ID = 'fin_search.product_service';
 
+    /** @var Config|null */
+    private $config;
```

Comment:
How could this ever be `null`? If it would be, the container should take care of the resolution (`$container->get(Config::class)`).